mix.exs

```diff
@@ -1,51 +1,85 @@
 defmodule TzWorld.Mixfile do
   use Mix.Project
 
+  @version "0.2.0"
+
   def project do
     [
       app: :tz_world,
       name: "TzWord",
-      version: "0.1.0",
-      elixir: "~> 1.3",
-      start_permanent: Mix.env == :prod,
+      version: @version,
+      elixir: "~> 1.8",
+      start_permanent: Mix.env() == :prod,
       deps: deps(),
+      docs: docs(),
       source_url: "https://github.com/kimlai/tz_world",
       description: description(),
       package: package(),
-      docs: [
-        extras: ["README.md": [title: "README"]],
-        main: "readme",
-      ],
+      dialyzer: [
+        plt_add_apps: ~w(mix inets jason)a
+      ]
     ]
   end
 
   # Run "mix help compile.app" to learn about applications.
   def application do
     [
       mod: {TzWorld.Application, []},
-      extra_applications: [:logger, :ecto]
+      extra_applications: [:logger]
     ]
   end
 
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:geo, "~> 1.0"},
-      {:ex_doc, "~> 0.16", only: :dev, runtime: false},
+      {:geo, "~> 3.0"},
```

Comment:
I'm not sure that we should impose this version. I believe that the code works with any version of `geo` from 1.x to 3.x.